ansible/prepare_host_env.sh

```diff
@@ -0,0 +1,36 @@
+if [ "$EUID" -ne 0 ]
+	then echo "Please run this setup as root"
+	return
+fi
+
+if [[ "${BASH_SOURCE[0]}" == "${0}" ]]
+	then echo "Please run this setup with 'source' command"
+	exit
+fi
+
+
+### Install required packages
+echo "Installing required packages..."
+
+# git
+apt install git -y
```

Comment:
Swan is not supporting `Debian`'s

Reply:
Yet it can be build on it.

Reply:
It can be built b/c golang doesn't care about it. Swan in it's basics doesn't support Debians ;P

Reply:
As far as I understand this script is to be run on developer's machine. OS is not very relevant then.